app/scripts/modules/core/src/pipeline/config/stages/managed/publishDeliveryConfigStage.ts

```diff
@@ -0,0 +1,24 @@
+import { Registry } from 'core/registry';
+import { ExecutionDetailsTasks } from 'core/pipeline';
+
+import { PublishDeliveryConfigStageConfig } from './PublishDeliveryConfigStageConfig';
+import { PublishDeliveryConfigExecutionDetails } from './PublishDeliveryConfigExecutionDetails';
+import { IStageOrTriggerBeforeTypeValidationConfig } from 'core/pipeline/config/validation/stageOrTriggerBeforeType.validator';
+
+Registry.pipeline.registerStage({
+  label: 'Publish Delivery Config',
+  description: 'Publish Delivery Config Manifest',
+  key: 'publishDeliveryConfig',
+  restartable: false,
+  component: PublishDeliveryConfigStageConfig,
+  executionDetailsSections: [PublishDeliveryConfigExecutionDetails, ExecutionDetailsTasks],
+  //validateFn: validate,
```

Comment:
I talked to a few people on the UI team; the preferred approach for validation is now to use `validateFn` instead of `validators`. There are a few [examples here](https://github.com/spinnaker/deck/search?q=validateFn&unscoped_q=validateFn).

Reply:
I couldn't quite figure out if it would be possible to reuse the existing "trigger-or-stage before" validator implementation with the `validateFn` interface, and it seemed non-critical to solve in this PR, so I'd propose a UI engineer take that up as a follow-up PR, if that's OK. Unless there's something obvious that I may well be missing that would allow me to reuse that -- please let me know.